doc/extdev/projectapi.rst

```diff
@@ -0,0 +1,9 @@
+.. _project-api:
```

Comment:
OMG, this file is orphaned!

Reply:
This comment was my wrong. This is surely referenced from `doc/extdev/index.rst`. np.